src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.dao.ProjectDaoImpl;
+import core.basesyntax.impl.CsvFileReaderServiceImpl;
+import core.basesyntax.impl.CsvFileWriterServiceImpl;
+import core.basesyntax.impl.FruitServiceImpl;
+import core.basesyntax.impl.ParserServiceImpl;
+import core.basesyntax.impl.ReportServiceImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvFileReaderService;
+import core.basesyntax.service.CsvFileWriterService;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.ParserService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class Main {
+    private static final String INPUT_FILE = "src/StoreInformation.csv";
+    private static final String WRITE_FILE = "src/Report.csv";
```

Comment:
as described in common mistakes let's place the input and output files into the src/main/resources folder